robocop/reports/json_report.py

```diff
@@ -6,14 +9,63 @@ class JsonReport(Report):
     """
     Report name: ``json_report``
 
+    Report that returns list of found issues in JSON format. The output file will be generated
+    in the current working directory with the ``robocop_output.json`` name.
+
+    This report is not included in the default reports. The ``--reports all`` option will not enable this report.
+    You can still enable it using report name directly: ``--reports json_report`` or ``--reports all,json_report``.
+
+    You can configure output directory and report filename::
+
+        robocop --configure json_report:output_dir:C:/json_reports
+        robocop --configure json_report:report_filename:robocop.json
+
+    """
+
+    DEFAULT = False
+
+    def __init__(self):
+        self.name = "json_report"
+        self.description = "Accumulates found issues in JSON format"
+        self.output_dir = None
+        self.report_filename = "robocop_output.json"
```

Comment:
Maybe just `robocop.json`? Do we have any other use of JSON files in relation to Robocop which may conflict here?

Reply:
No,  there shouldn't be conflict. I also used robocop.json initially, not sure why I renamed it but I will revert it, sounds better.